src/universal/modules/userDashboard/components/UserActionList/UserActionList.js

```diff
@@ -122,6 +131,8 @@ const styleThunk = () => ({
     borderWidth: '0 1px 1px',
     boxShadow: `inset 0 1px 0 ${ui.cardBorderColor}`,
     maxHeight: '100%',
+    // @terry this is required for no scroll bars, but conflicts with overflowTouch
```

Comment:
@ackernaut ah, just seeing this. Originally, i saw both x & y scrollbars while a card was being dragged. whatever magic you did, that no longer happens. thanks!